db/model/Gdoc/Gdoc.ts

```diff
@@ -284,44 +282,51 @@ export class Gdoc extends BaseEntity implements OwidGdocInterface {
     ): Promise<void> {
         const slugToIdMap = await Chart.mapSlugsToIds()
         const uniqueSlugsByLinkType = this.links.reduce(
-            (slugsByLinkType, { linkType, target }) => {
+            (slugsByLinkType, { linkType, target, queryString }) => {
```

Comment:
Taking query string into account here deduplicates a bit too much, no? For loading purposes we only care about the target - or am I missing something? In practice it's unlikely that a single doc would link to the same chart dozens of times but if a chart is used more than once we incur at least the DB round trip again for no gain. Maybe at least add a comment here about this?

Reply:
On the other hand, it might come in handy when we decide to create static svg exports of a chart which take the query params into account, as we currently do for WP posts (but don't currently do for gdocs posts afaik).